src/_blog/libp2p-webrtc-browser-to-server.md

```diff
@@ -0,0 +1,248 @@
+
+### Usage
+
+The complexity of WebRTC is abstracted in the implementations, making it seamless to swap your existing transport with WebRTC.
+
+Let's look at the JavaScript implementation as an example:
+
+```javascript
+import { webRTC } from 'js-libp2p-webrtc'
+
+const node = await createLibp2p({
+  transports: [webRTC()],
+  connectionEncryption: [() => new Noise()],
+});
+```
+
+The only difference from other transports is initializing with `webRTC()`.  That's all you need to do to implement WebRTC in the browser.  Easy, right?
+
+
+## Alternative transports
+
+WebRTC is one of many ways to connect browsers to a libp2p node. [Choosing the transport](https://connectivity.libp2p.io/) that fits your use case is one of the many unique strengths of libp2p.
+
+### WebSocket
+
+The [WebSocket RFC](https://datatracker.ietf.org/doc/html/rfc6455) dates back to 2011 and specifies the opening of a two-way socket from a browser to a server over TCP. WebSocket is implemented in the [Rust](https://github.com/libp2p/rust-libp2p/tree/master/transports/websocket), [Go](https://github.com/libp2p/go-libp2p/tree/master/p2p/transport/websocket), and [JavaScript](https://github.com/libp2p/js-libp2p-websockets) libp2p implementations. 
+
+#### Limitations
+
+The various upgrades and handshakes add up to six round trips before data can be exchanged. Additionally, while WebRTC can leverage self-signed certificates, WebSockets cannot, as they require the server to have a trusted TLS certificate using TCP.
+
+### WebTransport
+
+[WebTransport](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-overview) is the new kid on the block for real-time communication in the browser.  WebTransport is implemented in [Go](https://github.com/libp2p/go-libp2p/tree/master/p2p/transport/webtransport) and [JavaScript](https://github.com/libp2p/js-libp2p-webtransport) implementations. 
+
+#### Benefits
+
+WebTransport has many of the enhanced features of WebRTC (fast, secure, multiplexed) without requiring servers to implement the stack while also getting around the valid TLS certificate requirement.
+
+As opposed to WebSockets, libp2p can use raw WebTransport streams and avoid the need for double encryption.  
+
+WebTransport requires less roundtrips than WebRTC to establish a connection, making it the preferred choice when supported.
+
+#### Limitations
+
+You might be asking yourself, why pick WebRTC over WebTransport in libp2p? It's like WebRTC but easier to implement and with less complexity. WebTransport is not without its limitations.
+
+WebTransport isn't supported [in all browsers](https://caniuse.com/webtransport). This lack of support is a big concern as it's unreasonable to expect users to _not_ use Firefox or Safari, or even older versions of Chromium-based browsers.  While both Firefox and Safari plan to implement WebTransport, WebRTC is a great fallback while we wait for universal support.
```

Comment:
Practically nobody is using a 2-digit Chrome version any more. I suggest the following rewording:
```suggestion
The WebTransport protocol itself is still under development, and currently only implemented supported [in Chrome](https://caniuse.com/webtransport), but not yet in Firefox or Safari.  Until WebTransport is implemented by all major browsers, WebRTC is a great fallback.
```